app/controllers/concerns/sufia/file_sets_controller_behavior.rb

```diff
@@ -5,7 +5,7 @@ module FilesController
     autoload :LocalIngestBehavior
```

Comment:
@mjgiarlo yeah, I wondered about that too... this also prompted my question in #1424. I would go with a follow up ticket and perhaps more discussion on the issue above.